concent_api/concent_golem_messages/tests/factories.py

```diff
@@ -0,0 +1,69 @@
+import random
+
+from ethereum.utils import denoms
+from golem_messages.factories.helpers import MessageFactory
+import factory.fuzzy
+
+from concent_golem_messages.message import NonceAbstractMessage
+from concent_golem_messages.message import SignedTransaction
+from concent_golem_messages.message import TransactionRejected
+from concent_golem_messages.message import TransactionSigningRequest
+
+
+class NonceAbstractMessageFactory(MessageFactory):
+    class Meta:
+        model = NonceAbstractMessage
+
+    nonce = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+
+
+class TransactionAbstractMessage(NonceAbstractMessageFactory):
+    class Meta:
+        model = NonceAbstractMessage
+
+    gasprice = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+    startgas = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+    to = factory.fuzzy.FuzzyText(length=20, chars='0123456789abcdef')
+    value = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+    data = b''
```

Comment:
@kbeker https://github.com/golemfactory/concent/pull/687#discussion_r207898326
> Please add tests which will raise exceptions if gasprice, startgas, to and value will be bigger than our assumption

:+1: done, but tests are in `test_message.py`.